jobs/build-triggers/tendrl-build-triggers.yml

```diff
@@ -201,4 +211,5 @@
       - "tendrl-build-1-{project_name}-monitoring-integration"
       - "tendrl-build-1-{project_name}-node-agent"
       - "tendrl-build-1-{project_name}-notifier"
+      - "tendrl-build-1-{project_name}-tendrl-ansible"
```

Comment:
you'll also require below line

`tendrl-build-1-{project_name}-tendrl-selinux`